public/app/index.html

```
@@ -288,6 +288,72 @@ <h1 class="hdr-title">VoiceIsolate Pro <span class="hdr-ver">v25.0</span></h1>
             <span class="tp-ab-tag">A</span>
             <span class="tp-ab-name">Original</span>
           </span>
+<<<<<<< HEAD
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)